content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+As solicitações HTTP e HTTPS para o Ingress que correspondem ao host e ao caminho da regra são enviadas para o backend listado.
+
+Um `defaultBackend` geralmente é configurado em um controlador Ingress para atender a quaisquer solicitações que não correspondam a um caminho na especificação.
+
+### DefaultBackend {#default-backend}
+
+Um Ingress sem regras envia todo o tráfego para um único backend padrão e `.spec.defaultBackend` é o backend que deve lidar com as solicitações nesse caso. 
+O `defaultBackend` é convencionalmente uma opção de configuração do [controlador Ingress](/docs/concepts/services-networking/ingress-controllers) e não é especificado em seus recursos Ingress. 
+Se nenhum `.spec.rules` for especificado, o `.spec.defaultBackend` deve ser especificado. 
+Se o `defaultBackend` não for definido, o tratamento de solicitações que não correspondem a nenhuma das regras ficará a cargo do controlador de Ingress (consulte a documentação do seu controlador de Ingress para descobrir como ele lida com esse caso).
+
+Se nenhum dos hosts ou caminhos corresponder à solicitação HTTP nos objetos Ingress, o tráfego será roteado para o seu backend padrão.
+
+### Resource backends {#resource-backend}
+
+Um `Resource` backend é um ObjectRef para outro recurso Kubernetes dentro do mesmo namespace que o objeto Ingress. 
+Um `Resource` é uma configuração mutuamente exclusiva com o serviço, e a validação irá falhar se ambos forem especificados. 
+Um uso comum para um `Resource` backend é inserir dados em um backend de armazenamento de objetos com ativos estáticos.
+
+{{< codenew file="service/networking/ingress-resource-backend.yaml" >}}
+
+Depois de criar o Ingress acima, você pode visualizá-lo com o seguinte comando:
+
+```bash
+kubectl describe ingress ingress-resource-backend
+```
+
+```
+Name:             ingress-resource-backend
+Namespace:        default
+Address:
+Default backend:  APIGroup: k8s.example.com, Kind: StorageBucket, Name: static-assets
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *
+              /icons   APIGroup: k8s.example.com, Kind: StorageBucket, Name: icon-assets
+Annotations:  <none>
+Events:       <none>
+```
+
+### Tipos de caminho
+
+Cada caminho no Ingress deve ter um tipo de caminho correspondente. 
+Os caminhos que não incluem um `pathType` explícito falharão na validação. 
+Existem três tipos de caminho suportados:
+
+* `ImplementationSpecific`: Com esse tipo de caminho, a correspondência depende da IngressClass. As implementações podem tratar isso como um `pathType` separado ou tratá-lo de forma idêntica aos tipos de caminho `Prefix` ou `Exact`.
+* `Exact`: Corresponde exatamente ao caminho da URL e com _case-sensitive_.
```

Comment:
Hum, I can't remember if there is a real definition that Exact is case sensitive: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#httpingresspath-v1-networking-k8s-io

Per API, there's no such specification, so I would rather say that "may be case-sensitive" depending on your ingress controller specification (unless you can find a spec that says otherwise! Because I couldn't and can't remember if we follow this pattern on ingress nginx tbh)